gitlab/data_source_gitlab_projects.go

```diff
@@ -850,3 +850,19 @@ func dataSourceGitlabProjectsRead(d *schema.ResourceData, meta interface{}) (err
 	}
 	return err
 }
+
+func flattenSharedWithGroupsOptions(project *gitlab.Project) []interface{} {
```

Comment:
This function is moved from its original home in `gitlab/resource_gitlab_project.go`.